website/src/pages/pricing.tsx

```diff
@@ -48,120 +72,152 @@ export default ((props: any) => (
             <div className="pricing-page mt-2">
                 <ContentSection className="hero-section text-center py-5">
                     <h1 className="display-2 font-weight-bold">Sourcegraph Pricing</h1>
-                    <h2>Universal Code Search scales with your team</h2>
+                    <h2>Universal Code Search</h2>
                 </ContentSection>
                 <div className="container-fluid pricing-page__plans">
                     <div className="row pt-4">
-                        <div className="col-md-4 mx-auto mb-4">
+                        <div className="col-md-6 mx-auto mb-4">
                             <PricingPlan
                                 className="pricing-page__plan"
                                 name="Free"
                                 price={<div className="text-center">$0/mo</div>}
                                 planProperties={
                                     <>
-                                        <PricingPlanProperty
-                                            description={
-                                                <>
-                                                    <br />
-                                                </>
-                                            }
-                                        >
-                                            Includes 10 users
+                                        <PricingPlanProperty>
+                                            Up to 10 users
+                                        </PricingPlanProperty>
+                                        <PricingPlanProperty className="mt-3">
+                                            Community support<br/>&nbsp;
                                         </PricingPlanProperty>
-                                        <PricingPlanProperty className="mt-3">Community support</PricingPlanProperty>
```

Comment:
@jeanduplessis - can we change this to "Community support on our public issue tracker" instead of just "Community support."